### ISSUE TITLE
Upstream changes?

### ISSUE THREAD

**Issue by Eeems** [slotThe/RemarkableLamyEraser#3]
I'm a little confused as to why you chose to fork instead of upstreaming the changes?

**Comment by slotThe** [comment]
Maybe I read https://github.com/isaacwisdom/RemarkableLamyEraser/issues/70 wrong, but my impression was that @isaacwisdom did not have time for maintenance of any kind anymore (which I also gather from the default branch being switched to `v1`). I'd be glad to upstream this if I could perhaps get a commit bit to the original repository (e.g., the repo could go to some kind of organisation), so iteration times could be held to a minimum.

**Comment by Eeems** [comment]
It would probably be worth asking to get access to the repo. It doesn't need to be part of an organization to add more contributors with access.

**Comment by isaacwisdom** [comment]
I'm happy to give someone access to the project. Is it possible to make the project "open", in the sense that anyone can make commits, and it's up to the community to keep the project maintained?

**Comment by Eeems** [comment]
> I'm happy to give someone access to the project. Is it possible to make the project "open", in the sense that anyone can make commits, and it's up to the community to keep the project maintained?

No, you can't make it open, but you can have multiple people as contributors that can merge pull requests.

**Comment by slotThe** [comment]
@isaacwisdom thanks for giving me access to the repository! It seems like I need juuust a few more rights, as right now I—for example—can't change the default branch

**Comment by slotThe** [comment]
@isaacwisdom friendly ping :)

**Comment by isaacwisdom** [comment]
Hey sorry for the delay on this @slotThe. I looked into it and I wasn't able to see how to give you permission to change the default branch. That being said, would you like me to switch the branch back to v2?

**Comment by slotThe** [comment]
@isaacwisdom I should already be listed under `Settings -> Collaborators`, right? You should then be able to set my "role"; I think the only one that has enough permissions is "Admin".

> That being said, would you like me to switch the branch back to v2?

I have created a `main` branch, this is the one I'd like to make default :)
